### DISCUSSION TITLE
Support for multiple languages ​​?

### DISCUSSION THREAD

**Discussion by IGNOR90** [rafaelurben/django-discordbot#225] [Ideas]
Can you add support for multiple languages, or at least make the translation a bit easier?

**Comment by rafaelurben** [comment]
@IGNOR90 Hey! Thanks for your message! 

This project is currently on hold as I have some important things going on in my personal life at the moment. If I find time to finally create v2 of django-discordbot, this is certainly a feature to consider. 👍
